<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ignitors</class>
 <widget class="QWidget" name="Ignitors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="mainLayout" stretch="6,16,1">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="LCDLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="pressureLCDFrame">
         <property name="styleSheet">
          <string notr="true">#pressureLCDFrame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} </string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="pressureLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Black</family>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Pressure [Bar]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="pressureLCD">
            <property name="styleSheet">
             <string notr="true">QLCDNumber
{
}

QFrame
{
	border: 0px, white;
	border-radius: 25px;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="tempLCDFrame">
         <property name="styleSheet">
          <string notr="true">#tempLCDFrame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} </string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="tempLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Black</family>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Temperature [ÂºC]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="tempLCD">
            <property name="styleSheet">
             <string notr="true">

QFrame
{
	border: 0px;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="currentLCDFrame">
         <property name="styleSheet">
          <string notr="true">#currentLCDFrame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} </string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="currentLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Black</family>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Current [A]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="currentLCD">
            <property name="styleSheet">
             <string notr="true">

QFrame
{
	border: 0px;
	border-radius: 25px;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="graphLayout" stretch="1,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="pressureFrame">
         <property name="styleSheet">
          <string notr="true">QFrame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCustomPlot" name="pressureGraph" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="currentFrame">
         <property name="styleSheet">
          <string notr="true">QFrame
{
	border-radius: 30%;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	
} 

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QCustomPlot" name="currentGraph" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { border-radius: 20%; background-color: white; }</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
